hw_diag/utilities/miner.py

```diff
@@ -8,9 +9,21 @@ def fetch_miner_data(diagnostics):
     try:
         peerbook = client.get_peer_book()[0]
         height = client.get_height()
+        # Used for unit test to make sure exception
+        # can be thrown from hm-pyhelper
+        if "MC" in diagnostics and diagnostics['MC'] == 'testexception':
```

Comment:
Is this code intentionally left in?

Reply:
This is a unit test to make sure the exception can be thrown in the first place and that a valid connection to hm-pyhelper exists. diagnositcs['MC'] will = 'test exception' only when called from the unit test

Reply:
Removed it as I was using side effect